site/source/docs/compiling/Modularized-Output.rst

```diff
@@ -0,0 +1,136 @@
+.. _Modularized-Output:
+
+==================
+Modularized Output
+==================
+
+By default, emscripten outputs regular JS scripts which can be loaded the Web via
+a ``script`` tag, or o the command line using node or some other JS runtime.
+
+In this default configuration thw generated :ref:`module` variable (and indeed
+all of the emscripten internal symbols) are added to the global scope.  This
+means that only one emscripten-build program can be loaded in a given global
```

Comment:
```suggestion
means that only one emscripten-built program can be loaded in a given global
```

Reply:
I see you asked for review again, but some typo fixes remain pending, like this one?

Reply:
Oops, somehow missed that one.  Done